### PR TITLE
chore(PropTypes): Fix proptypes for new RN

### PR DESCRIPTION
**Pull request by bayukp**
`Image.propTypes.source` is not available in newer RN.
Change to `PropTypes.object`

### CODE REVIEW

File: elements/Image.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default class extends Shape {
         y: numberProp,
         width: numberProp.isRequired,
         height: numberProp.isRequired,
-        href: Image.propTypes.source,
+        href: PropTypes.object,
```
Comment (maxkomarychev) [inline elements/Image.js]: it's not very accurate, source can also be a `number` when image is `require`d
https://github.com/facebook/react-native/blob/master/Libraries/Image/ImageSourcePropType.js#L82-L88

const ImageSourcePropType = PropTypes.oneOfType([
  ImageURISourcePropType,
  // Opaque type returned by require('./image.jpg')
  PropTypes.number,
  // Multiple sources
  PropTypes.arrayOf(ImageURISourcePropType),
]);

**Comment by msand** [comment]
Hmm, I'm not having this problem, are you sure this is an issue?

**Comment by bayukp** [comment]
@msand build with js prod enable will crash your app.

File: elements/Image.js (hunk 1 above)
Comment (bayukp) [reply elements/Image.js]: will update accordingly.

**Comment by maxkomarychev** [comment]
regarding original issue with prod build: @bayukp what is your react-native?
Also there's actually special [`ImageSourcePropType`](https://github.com/facebook/react-native/blob/master/Libraries/Image/ImageSourcePropType.js) can be imported from `react-native`

**Comment by bayukp** [comment]
@maxkomarychev I'm using 0.54.

Yes i will update using that one.

But since i doesn't have access to my repo anymore, i will create separate PR.

**Comment by bayukp** [comment]
move to https://github.com/react-native-community/react-native-svg/pull/722

**Pull request closed without merging**